src/System.Windows.Forms/tests/UnitTests/System/Windows/Forms/DataGridViewCellConverterTests.cs

```diff
@@ -0,0 +1,54 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+#nullable enable
+
+using System.ComponentModel.Design.Serialization;
+using System.Globalization;
+
+namespace System.Windows.Forms.Tests;
+
+public class DataGridViewCellConverterTests
+{
+    private readonly DataGridViewCellConverter _converter = new();
+
+    [WinFormsTheory]
+    [InlineData(typeof(InstanceDescriptor), true)]
+    [InlineData(typeof(string), true)]
+    public void CanConvertTo_ReturnsExpected(Type destinationType, bool expected)
+    {
+        _converter.CanConvertTo(null, destinationType).Should().Be(expected);
+    }
```

Comment:
We prefer expression-bodied syntax - 
```suggestion
    public void CanConvertTo_ReturnsExpected(Type destinationType, bool expected) =>
        _converter.CanConvertTo(null, destinationType).Should().Be(expected);
```